src/emmetHelper.ts

```diff
@@ -5,17 +5,17 @@
 
 
 import { TextDocument, Position, Range, CompletionItem, CompletionList, TextEdit, InsertTextFormat, CompletionItemKind } from 'vscode-languageserver-types'
-import { expand, createSnippetsRegistry, parse } from './expand/expand-full';
-import * as extract from '@emmetio/extract-abbreviation';
+import { expand, parse, extract } from './emmetCompat';
 import * as path from 'path';
 import * as fs from 'fs';
 import * as JSONC from 'jsonc-parser';
 import { homedir } from 'os';
 import { cssData, htmlData } from './data';
+import { OldEmmetExpandOptions } from './oldEmmetTypes';
+import { resolveConfig } from 'emmet';
 
 const snippetKeyCache = new Map<string, string[]>();
 let markupSnippetKeys: string[];
-let markupSnippetKeysRegex: RegExp[];
```

Comment:
`RegEx -> string` snippets where never supported in the first place, because snippets were taken only from json file which had `string -> string` only. Hence almost everywhere you'll find only `snippetRegistry.all({ type: "string" })`.
And anyways the newer emmet doesn't support them too.